libraries/AP_HAL_F4BY/AP_HAL_F4BY_Namespace.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+namespace F4BY {
+	class F4BYScheduler;
+	class F4BYUARTDriver;
+	class F4BYStorage;
+	class F4BYRCInput;
+	class F4BYRCOutput;
+	class F4BYAnalogIn;
+	class F4BYAnalogSource;
+	class F4BYUtil;
+        class F4BYGPIO;
+        class F4BYDigitalSource;
+        class NSHShellStream;
+        class F4BYI2CDriver;
+        class F4BY_I2C;
+        class Semaphore;
+}
```

Comment:
This works, but every time this header is changed, it triggers unnecessary rebuild for a lot of cpp files. Also, see this commit, which has already been merged, https://github.com/ArduPilot/ardupilot/pull/4574/commits/f36e8d9c0559ae7214a9d4c5c9b1ca06209ba390.